stats/bin/spark-stats.js

```diff
@@ -17,7 +18,7 @@ const logger = {
   request: ['1', 'true'].includes(REQUEST_LOGGING) ? console.info : () => {}
 }
 
-const handler = createHandler({ pgPools, logger })
+const handler = createHandler({ SPARK_API_BASE_URL, pgPools, logger })
```

Comment:
in this PR, `SPARK_API_BASE_URL` is never overridden, so it should be removed from args and added to config instead

Reply:
I want to make this configurable via env vars so we can run spark-stats locally with redirects to the local spark-api instance. With your proposal, I would have to edit the local config file and then remember to discard the changes before committing to git.

In that light, do you still prefer to move this to the config file? I don't have a strong opinion.

Reply:
Got it, I didn't see that possibility. I never run these services in dev 😅